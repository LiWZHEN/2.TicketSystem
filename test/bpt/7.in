56
insert a 12
insert b 12
insert c 12
insert d 12
insert e 12
insert f 12
insert g 12
insert h 12
insert i 12
insert j 12
insert k 12
insert l 12
insert m 12
delete a 12
delete b 12
delete c 12
delete d 12
delete e 12
delete f 12
delete g 12
delete h 12
delete i 12
delete j 12
delete k 12
find m
delete l 12
delete m 12
find m
insert a 12
insert a 12
insert a 12
insert b 12
insert c 12
insert d 12
insert e 12
insert f 12
insert g 12
insert h 12
insert i 12
insert j 12
insert k 12
insert l 12
insert m 12
find a
find b
find c
find d
find e
find f
find g
find h
find i
find j
find k
find l
find m
