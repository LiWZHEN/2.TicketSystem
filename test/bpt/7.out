12 
null
12 
12 
null
12 
12 
12 
12 
12 
12 
12 
12 
12 
12 
